services/contrib/FactsServiceAgent/README.rst

```diff
@@ -0,0 +1,83 @@
+Ecorithm's Facts Service Agent
+==============================
+Ecorithm's Facts Service agent is used to push BACnet data to a remote cloud storage solution at frequent intervals.
```

Comment:
Historians are protocol agnostic and don't require BACnet or anything else specific in terms of data protocol. References to BACnet in the docs should be removed.